src/vs/workbench/browser/layout.ts

```diff
@@ -339,6 +346,15 @@ export class WorkbenchLayout implements IVerticalSashLayoutProvider, IHorizontal
 
 		// Activity Bar
 		let activityBarMinWidth = this.computedStyles.activitybar.minWidth;
+
+		// For inline titles, we want to ignore the minWidth on the activity bar
+		// and generate a zoom-dependent width, to ensure all other parts lay out correctly
+		const windowConfig = this.configurationService.getConfiguration<IWindowConfiguration>();
+		if (windowConfig && windowConfig.window.macOSUseInlineToolbar) {
+			const zoom = webFrame.getZoomFactor();
+			activityBarMinWidth = 76/zoom;
```

Comment:
@orta the 76px should be read from the CSS file if possible and not hardcoded (compare with https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/browser/layout.ts#L307)